app/src/main/res/layout/album_list_item.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android" android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <LinearLayout
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        android:gravity="center"
+        android:padding="20dp">
+
+    <ImageView android:id="@+id/album_list_artwork"
+        android:layout_width="100dp"
+        android:layout_height="100dp"
+        android:src="#636363"/>
+
+    <TextView android:id="@+id/album_list_title"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:paddingTop="7dp"
+        android:text="UNKNOWN"/>
+
+    <TextView android:id="@+id/album_list_artist"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:paddingTop="7dp"
+        android:text="UNKNOWN"/>
```

Comment:
nit - space before `/>` in self closing elements, also change the other ones used in other xml layouts.